fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -378,8 +379,16 @@ public PlanFragment visitPhysicalHashJoin(PhysicalHashJoin<Plan, Plan> hashJoin,
                 .map(e -> ExpressionTranslator.translate(e, context))
                 .collect(Collectors.toList());
 
+        List<Expr> otherJoinConjuncts = hashJoin.getOtherJoinCondition()
+                .map(ExpressionUtils::extractConjunction)
+                .orElseGet(Lists::newArrayList)
+                .stream()
+                .filter(e -> !(e.equals(BooleanLiteral.TRUE)))
```

Comment:
do we have to consider const conjuncts here?
`TRUE`  is a special case of const conjunct. 
There are some other conjucts, like `1=1` or `FALSE`.
I think there should be a rule to clean them before translate.
So I think we do not need to filter them here.

Reply:
> do we have to consider const conjuncts here? `TRUE` is a special case of const conjunct. There are some other conjucts, like `1=1` or `FALSE`. I think there should be a rule to clean them before translate. So I think we do not need to filter them here.

It's should be consider by other rule

Reply:
So, filter(...) is useless. It is better to remove this call

Reply:
Tmp way, add TODO is enough.

Reply:
@jackwener say all what i want say

Reply:
add TODO